project1/server/server_tasks/command_ids.h

```diff
@@ -0,0 +1,24 @@
+/**
+ * @File The thread safe data structure to store active command ID's
+ */
+
+#ifndef PROJECT1_COMMAND_IDS_H
+#define PROJECT1_COMMAND_IDS_H
+
+#include <mutex>
+#include <algorithm>
+#include <list>
+namespace server_tasks {
+    class CommandIDs {
```

Comment:
This whole class is missing docs (and the formatting is weird).

Reply:
the `CMD-ALT-L` keybinding for formatting does this automatically. I've tried to look into changing this but I wasn't able to get it to not indent after namespace blocks...